src/common.props

```diff
@@ -0,0 +1,19 @@
+<Project>
+  <PropertyGroup>
+    <TargetFramework>netstandard2.1</TargetFramework>
+    <LangVersion>latest</LangVersion>
+    <Version>0.1.1</Version>
+    <authors>Darren</authors>
+    <owners>Darren</owners>
+    <description>dotnet wechaty</description>
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+    <PackageiconUrl>http://qn.3658k.com/dotnet-wechaty.png</PackageiconUrl>
```

Comment:
shall we change ```PackageiconUrl``` to [../docs/images/dotnet-wechaty.png](https://raw.githubusercontent.com/wechaty/dotnet-wechaty/master/docs/images/dotnet-wechaty.png)?

Reply:
PackageiconUrl  next edition  update